backend/app/notify/slack.go

```diff
@@ -0,0 +1,117 @@
+package notify
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	log "github.com/go-pkgz/lgr"
+	"github.com/pkg/errors"
+	"github.com/slack-go/slack"
+)
+
+// Slack implements notify.Destination for Slack
+type Slack struct {
+	channelID string
+	client    *slack.Client
+}
+
+// NewSlack makes Slack bot for notifications
+func NewSlack(token, channelName string) (*Slack, error) {
+	return newSlack(token, channelName, 0, "")
+}
+
+func newSlack(token, channelName string, _ time.Duration, apiURL string) (*Slack, error) {
+
+	if channelName == "" {
+		channelName = "general"
+	}
+
+	opts := []slack.Option{}
+	if apiURL != "" {
+		opts = append(opts, slack.OptionAPIURL(apiURL))
+	}
+
+	client := slack.New(token, opts...)
+	res := &Slack{client: client}
+
+	channelID, err := res.findChannelIDByName(channelName)
+	if err != nil {
+		return res, err
+	}
+
+	res.channelID = channelID
+	log.Printf("[DEBUG] create new slack notifier for chan %s", channelID)
+
+	return res, nil
+}
+
+// Send to Slack channel
+func (t *Slack) Send(ctx context.Context, req Request) error {
+
+	if t.channelID == "" {
+		return errors.New("client is not connected")
+
+	}
+	log.Printf("[DEBUG] send telegram notification, comment id %s", req.Comment.ID)
+
+	author := req.Comment.User.Name
+	if req.Comment.ParentID != "" {
+		author += " → " + req.parent.User.Name
```

Comment:
It's not an author actually, isn't it? I think that variable should be named differently.